scanners/flutter/flutter.go

```diff
@@ -252,28 +251,8 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
 		}
 		cfg := configName + "-" + testKey
 
-		if project.hasIosProject {
```

Comment:
Remove this entire if block because the iOS related options are not needed anymore